src/foam/core/reflow/PropertyOptionCitationView.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.reflow',
+  name: 'PropertyOptionCitationView',
+  extends: 'foam.u2.View',
```

Comment:
```suggestion
  extends: 'foam.u2.CitationView',
```
If you do this you can delete most of the css

Reply:
even when I try I wouldn't have to delete code this time, FOAM hits again🤣